docs/guides/advanced/schemas.md

```diff
@@ -459,6 +459,112 @@ In your sync formula you only need to populate the fields of the reference objec
     Reference schemas are only resolved to rows when they are used in a sync table. If used in a formula or column format they will always appear in a broken state, even if the row they are referencing is present.
 
 
+## Schemas in cards {: #cards}
+
+The contents of a card are defined using an object schema. The following fields are specific to cards:
+
+- [`titleProperty`][titleProperty]
+- [`subtitleProperties`][subtitleProperties]
+- [`snippetProperty`][snippetProperty]
+- [`linkProperty`][linkProperty]
+- [`imageProperty`][imageProperty]
+
+To be eligible to be displayed as a card, the schemas must meet all of the following criteria:
+
+--8<-- "guides/advanced/.card_schema.md"
+
+Read the [Cards guide][cards] for more information on how these fields are used.
+
+
+### Property labels {: #labels}
+
+A card's subtitle contains a set of properties, joined with a separator. Unlike other areas of the card where the property's value is shown, in the subtitle a label is shown instead. By default this label takes the form of `{name}: {value}`.
```

Comment:
maybe clearer to say
"By default this label takes the form of `{name}: {value}`." -> "By default, the label and value are displayed together as '{label}: {value}'

Reply:
also nit: "unlike other areas of the card where **only** the property's value is shown, a label is shown in the subtitle as well"

Reply:
Good call. I had originally thought that `label` had to be a template, but only today realized you can pass a label only. Done.

Reply:
great